### PR TITLE
gls 1.0 (new formula)

### PR DESCRIPTION
**Pull request by rs**
The gls command is a graphical ls for iTerm2

### CODE REVIEW

File: Library/Formula/gls.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Gls < Formula
+  desc "A graphical ls command for iTerm2"
```
Comment (bfontaine) [inline Library/Formula/gls.rb]: Please remove the indefinite article here.

File: Library/Formula/gls.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  homepage "https://github.com/rs/gls"
+  url "https://github.com/rs/gls/archive/1.0.tar.gz"
+  sha256 "d5cb69827f57fd61d354418b08ff8ebf854e08daaeebb08aaa5a9d3450c06a54"
+  depends_on :xcode => :build
```
Comment (bfontaine) [inline Library/Formula/gls.rb]: Please add an empty line above this one.

File: Library/Formula/gls.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  def install
+    xcodebuild "-project", "gls.xcodeproj", "SYMROOT=build"
+    bin.install "build/Release/gls"
+  end
+end
```
Comment (bfontaine) [inline Library/Formula/gls.rb]: Please add a `test do` block.

**Comment by rs** [comment]
Should be fixed

File: Library/Formula/gls.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+class Gls < Formula
+  desc "Graphical \"ls\" command for iTerm2"
+  homepage "https://github.com/rs/gls"
+  head "https://github.com/rs/gls/archive/master.zip"
```
Comment (mikemcquaid) [inline Library/Formula/gls.rb]: This should just be `https://github.com/rs/gls.git`

File: Library/Formula/gls.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+class Gls < Formula
+  desc "Graphical \"ls\" command for iTerm2"
```
Comment (mikemcquaid) [inline Library/Formula/gls.rb]: Just use `'` quotes.

Comment (mikemcquaid) [inline Library/Formula/gls.rb]: (for the inner quotes)

File: Library/Formula/gls.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/rs/gls"
+  head "https://github.com/rs/gls/archive/master.zip"
+  url "https://github.com/rs/gls/archive/1.0.tar.gz"
+  sha256 "2c8fcd7c964b8c9d62dd20cb7c719b736800f23c5aa9d657995f33cedb149f1b"
+
+  depends_on :xcode => :build
+
+  def install
+    xcodebuild "-project", "gls.xcodeproj", "SYMROOT=build"
+    bin.install "build/Release/gls"
```
Comment (dunn) [inline Library/Formula/gls.rb]: Looks like it's building, but failing on this line:
```
==> xcodebuild -project gls.xcodeproj SYMROOT=build
Build settings from command line:
    SYMROOT = build

=== BUILD TARGET gls OF PROJECT gls WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies
warning: no rule to process file '/tmp/gls20150908-91075-i4chse/gls-1.0/gls/main.swift' of type text for architecture x86_64

** BUILD SUCCEEDED **

Error: No such file or directory - build/Release/gls
```

**Comment by dunn** [comment]
There are also a few audit warnings:
```
==> audit problems
gls:
 * `checksum` (line 6) should be put before `head` (line 4)
 * HEAD: Use GitHub tarballs rather than zipballs (url is https://github.com/rs/gls/archive/master.zip).
 * GitHub repository too new (<30 days old)
```
It looks like it's gotten a bit of popularity, but it's still very new.  We're hesitant to take on the work of maintaining a formula that don't have a track record of consistent development, so we may leave this open for a few days at least.

**Comment by rs** [comment]
Ok fair enough. I will fix the different issue just in case you decide to include it.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
It's failing to build on all three platforms.

**Comment by bfontaine** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**